python/src/tasks/mitochondrial_content.py

```diff
@@ -1,14 +1,15 @@
 import json
 from config import get_config
 from result import Result
+import numpy as np
 import requests
 
 config = get_config()
 
 
 class GetMitochondrialContent:
-    def __init__(self, msg):
-        self.task_def = msg["body"]
+    def __init__(self):
+        self.task_def = "MT-content"
```

Comment:
Same here, as the task does not take any more parameter from `msg`, I think you can just delete this assignment.